06-getting-your-class-going-with-jupyter.md

```diff
@@ -9,52 +9,52 @@ internet.
+(or root) access to the computer. Anaconda includes over 1500 software packages
+providing most, if not all, needed software for learners. Jupyter notebooks can
+be opened by launching Jupyter or by opening them through the Spyder IDE. These
+attributes make it attractive for both personal use and for installation on
+institution controlled computers.
+
+>**What is Anaconda?** You will see the Anaconda distribution recommended by
+>many educators and course authors. Anaconda is a package manager, an
+>environment manager, a Python distribution, a collection of [over 1,500+ open
+>source packages](https://docs.anaconda.com/anaconda/packages/pkg-docs/), and
+>also Jupyter. It is free to download, open source, and easy to install, on any
+>computer system (Windows, Mac OS X or Linux). It also includes the conda
+>packaging utility to update and install new packages of the Python and R
+>ecosystems, and to manage computational environments. According to the
+>company's webpage, Anaconda has more than 6 million users:
+><https://www.anaconda.com/what-is-anaconda/>.  The Software Carpentry project
+>provides installation instructions with videos for anaconda:
 ><https://swcarpentry.github.io/python-novice-gapminder/setup/>.
 
-Two other popular open source projects that can run Jupyter notebooks
-are [nteract](https://nteract.io/) and
-[Hydrogen](https://nteract.io/atom). nteract is installed by
-downloading a binary installer from their website and double-clicking
-the installation file. Once nteract is installed, any Jupyter notebook
-on a student's local system can be double-clicked and it will open
-within nteract. nteract's simple user interface make it an excellent
-choice for students new to computer programming. Hydrogen is a [very
-popular plugin](https://atom.io/packages/hydrogen) for the open source
-Atom editor; it's currently used by over 700,000 people. Hydrogen lets
-a user edit, display, and execute a notebook within the Atom editor.
-
-You can ask students to install Jupyter on their own computer or make
-it possible for them to use lab computers. These can also be combined:
-give students the instructions to install it on their own, but also
-tell them that it's available in the lab if they can't get it to work
-on their laptop. This way you don't need a large enough computer lab
-for everyone, and don't need to worry that not everyone can get it to
-work on their own.
-
+Two other easily installable software packages that can run Jupyter notebooks
+are [nteract](https://nteract.io/) and [Hydrogen](https://nteract.io/atom).
+nteract is installed by downloading a binary installer from their website and
+double-clicking the installation file. nteract's simple user interface make it
+an excellent choice for students new to computer programming. Once nteract is
+installed, any Jupyter notebook on a student's local system can be
+double-clicked and it will open within nteract. Anaconda does not default to
```

Comment:
I don't think it is a double-click for every OS

Reply:
Double comments...

Reply:
A bit confused on the Anaconda sentence. nteract does allow double clicking on Mac, Windows, Linux to launch a notebook within nteract. Is the sentence about Anaconda saying its confusing that Anaconda does not allow double click to launch Jupyter? I would recommend striking the Anaconda sentence within the nteract section.